## Today, we will focus on the GORG-classifier for classifying meta'omic data using GORG-Dark genomes as a reference.

---
### Other approaches to integration of meta'omic and SAG data

---
#### Read Aligners

Nucleotide short read alignment
Allows you to have complete control over parameters to fine tune your analyses  

[bwa](https://bio-bwa.sourceforge.net/). 
[bbmap](https://jgi.doe.gov/data-and-tools/software-tools/bbtools/bb-tools-user-guide/bbmap-guide/)  
[bowtie2](https://bowtie-bio.sourceforge.net/bowtie2/manual.shtml)  

Protein-sequence-based  
[Diamond](https://github.com/bbuchfink/diamond)

Down-stream processing required! 
* e.g. [samtools](http://www.htslib.org/), [rsamtools](https://bioconductor.org/packages/release/bioc/html/Rsamtools.html)  



---
#### Wrappers  
[coverM](https://github.com/wwood/CoverM)  
[sag-mg-recruit](https://github.com/BigelowLab/sag-mg-recruit/tree/master)  

---
#### Read Classifiers  
[kraken2](https://ccb.jhu.edu/software/kraken/)  
[kaiju](https://github.com/bioinformatics-centre/kaiju)  

Efficient code, but only as good as the reference database.  

---
#### Graphical User Interfaces  

[Anvi'o](https://merenlab.org/tutorials/single-cell-genomics/)  

Pay for use:  
[Geneious](https://www.geneious.com/)  
[CLC Genomics Workbench](https://digitalinsights.qiagen.com/products-overview/discovery-insights-portfolio/qiagen-clc-genomics/?cmpid=QDI_GA_CLC&gclid=CjwKCAjwjMiiBhA4EiwAZe6jQxh5YJco-eviIbnR9rxBRq0rY8BRgkDAtBvEj9_xinXavzufSWdWwxoCtH8QAvD_BwE)  

